### DISCUSSION TITLE
Im-tired/

### DISCUSSION THREAD

**Discussion by giscus** [rpost/rpost.github.io#1] [General]
# Im-tired/

of low quality, complexity and bugs in software.

https://rpost.github.io/Im-tired/

**Comment by maciej-nowicki** [comment]
Sprzedam opla

**Reply by rpost** [reply]
jak tu się robi moderację? :thinking:

**Reply by rpost** [reply]
ogłoszenia bez ceny będą usuwane ;)

**Reply by maciej-nowicki** [reply]
Cena priv

**Comment by RG9** [comment]
nice, tidy blog template ;)
